ddtrace/contrib/django/patch.py

```diff
@@ -82,7 +89,10 @@ def cursor(django, pin, func, instance, args, kwargs):
             "django.db.alias": alias,
         }
         pin = Pin(service, tags=tags, tracer=pin.tracer, app=prefix)
-        return dbapi.TracedCursor(func(*args, **kwargs), pin, config.django)
+        traced_cursor_cls = (
```

Comment:
super nit (feel free to ignore): maybe makes more sense to structure it as a multiline if-statement since there could be other cursor cases in the future